src/main/java/de/focusshift/zeiterfassung/integration/urlaubsverwaltung/overtime/OvertimeRabbitmqConfiguration.java

```diff
@@ -0,0 +1,59 @@
+package de.focusshift.zeiterfassung.integration.urlaubsverwaltung.overtime;
+
+
+import de.focusshift.zeiterfassung.tenancy.tenant.TenantContextHolder;
+import org.springframework.amqp.core.Binding;
+import org.springframework.amqp.core.BindingBuilder;
+import org.springframework.amqp.core.Queue;
+import org.springframework.amqp.core.TopicExchange;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConditionalOnProperty(value = "zeiterfassung.integration.urlaubsverwaltung.overtime.enabled", havingValue = "true")
```

Comment:
Prefix zeiterfassung.integration.urlaubsverwaltung was used for incomming events by urlaubsverwaltung application. Maybe: `zeiterfassung.integration.overtime.publishing.enabled`